### ISSUE TITLE
Are you sure that you really need this function?

### ISSUE THREAD

**Issue by 00Programmer00** [iam047/homework5#1]
https://github.com/iam047/homework5/blob/b2a0a96c00aece8a36002fc55c1b150a647cbf99/js/app.js#L10
As for me this is needlessly

**Comment by iam047** [comment]
I agree, it would have been possible to do without it. Well, in this example, it was more convenient for me, in the following tasks I will be shorter code write
